Formula/azure-cli.rb

```diff
@@ -650,10 +645,6 @@ class AzureCli < Formula
   end
 
   def install
-    # Work around Xcode 11 clang bug
-    # https://code.videolan.org/videolan/libbluray/issues/20
-    ENV.append_to_cflags "-fno-stack-check" if DevelopmentTools.clang_build_version >= 1010
```

Comment:
Is this removal intentional?

Reply:
I think it is, as that seems like a bot.

Reply:
cc @azclibot

Reply:
@MyronFanQiu, who cut the 2.0.76 release commit, and works for Azure, and hopefully will either know the answer here, or knows who does. (And is human.)

(I'm just an onlooker who is interested in a feature in .76 and thus wants this PR.)

Reply:
@fengzhou-msft Hi. Could you take a look?

Reply:
@fxcoudert This removal is not intentional. We generate the formula by a template. I was not aware of this change here, so it's not in our template. Please add back here. I will add it in our template as well.